src/pages/overlay/App.tsx

```diff
@@ -93,6 +93,12 @@ export default function App() {
     wake(5000)
   }, [wake])
 
+  useEffect(() => {
+    if (sleeping) document.documentElement.style.cursor = 'none'
+    else document.documentElement.style.cursor = 'default'
+  }, [sleeping])
+
+
```

Comment:
```suggestion
  // Hide the cursor when sleeping
  useEffect(() => {
    if (sleeping) document.documentElement.style.cursor = 'none'
    else document.documentElement.style.removeProperty('cusor')
  }, [sleeping])

```
Few nitpicks here -- comment to describe what the effect is doing, using `removeProperty` to cleanup rather than setting it explicitly to `"default"`, and removing an extra linebreak